src/api/weatherApi.js

```diff
@@ -0,0 +1,31 @@
+import { USER, PASSWORD } from '../../env';
```

Comment:
as far as your env.js file not included in project directory (because of gitignore rules), anyone who will grab your code now, will see compilation error like "env not found". please fix it (google about  .env and how it works)